setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name='pyuntl',
-    version='1.0.2',
+    version='2.0.0',
     author='Mark Phillips',
```

Comment:
Lower in the file, to the `classifiers`, we could also add "'Programming Language :: Python :: 3.7',"

Reply:
we already have "Python :: 3.7" in setup.py classifiers. Did you mean to add "Python :: 3.6" ?

Reply:
yes, that is what I meant.